mempool/policy_test.go

```diff
@@ -21,6 +21,12 @@ import (
 	"github.com/decred/dcrd/wire"
 )
 
+const (
+	// noTreasury signifies the treasury agenda should be treated as though it
+	// is inactive.  It is used to increase the readability of the tests.
+	noTreasury = false
```

Comment:
would be great create this for other tests where `false` is being passed with the comment `// no treasury`.